lib/workers/repository/onboarding/branch/commit-message.ts

```diff
@@ -1,5 +1,6 @@
 import { RenovateConfig } from '../../../../config/types';
 import { CommitMessage } from '../../model/commit-message';
+import { CommitMessageFactory } from '../../model/commit-message-factory';
 
 export class OnboardingCommitMessageFactory {
```

Comment:
Looks like this class is a simple wrapper for `CommitMessageFactory` ? So it can removed in next PR?

Reply:
This class used in two places, therefore I would rather refactor it somehow. Basically it contains only logic about setting subject of commit message.